tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/klass/ClassTest.java

```diff
@@ -78,13 +78,8 @@ public void testRuntimeIsTypeCheckWithDistinctObjectTypes() {
     public void classDefNegative() {
         CompileResult negative = BCompileUtil.compile("test-src/klass/class_def_negative_test.bal");
         int i = 0;
-        String expectedErrorMsgPrefix = "invalid cyclic type reference in ";
-        BAssertUtil.validateError(negative, i++, expectedErrorMsgPrefix + "'[A, B, A]'", 17, 1);
-        BAssertUtil.validateError(negative, i++, expectedErrorMsgPrefix + "'[A, B, C, A]'", 17, 1);
-        BAssertUtil.validateError(negative, i++, expectedErrorMsgPrefix + "'[B, A, B]'", 22, 1);
-        BAssertUtil.validateError(negative, i++, expectedErrorMsgPrefix + "'[B, C, A, B]'", 22, 1);
-        BAssertUtil.validateError(negative, i++, expectedErrorMsgPrefix + "'[A, B, A]'", 28, 1);
-        BAssertUtil.validateError(negative, i++, expectedErrorMsgPrefix + "'[C, A, B, C]'", 28, 1);
+        BAssertUtil.validateError(negative, i++, "invalid cyclic type reference in 'A'", 17, 1);
```

Comment:
Is it possible to tell why it is cyclic, like before?

Reply:
Yes. Updated c64fd4e460acdf2c3f75e0794b4064834619cc8d